app/ports/tests/test_git_update.py

```diff
@@ -14,13 +15,15 @@ def test_between_new_old(self):
                                                      "ee0fc9c4ca59685f33628fc41c6946920869ab71",
                                                      TEST_REPO_PARENT
                                                      )
-        self.assertEquals(ports, ['ideviceinstaller', 'OpenBLAS', 'nomad'])
+        self.assertEquals(ports, ['ideviceinstaller', 'openblas', 'nomad'])
 
     def test_between_new_and_database(self):
-        git_update.get_list_of_changed_ports("7646d61853154b9dc523e3d0382960aea562e7ab",
-                                             "ee0fc9c4ca59685f33628fc41c6946920869ab71",
-                                             TEST_REPO_PARENT
-                                            )
+        if LastPortIndexUpdate.objects.count() > 0:
+            last_commit = LastPortIndexUpdate.objects.all().first()
```

Comment:
Would something like this work?
```
last_commit = LastPortIndexUpdate.objects.all().first()
if last_commit != nil
```
Or maybe even better: could `update_or_create()` work at this place?

We probably also need something to enforce a single entry at the database level. (Random link with an idea: https://stackoverflow.com/a/25393923, even if not so extremely important at this very point.)

Reply:
I have created a method `update_or_create_first_object` inside the Model class.